.travis.yml

```diff
@@ -16,20 +16,51 @@ jobs:
   - python: 3.9-dev
   - <<: *latest_py3
     env: TOXENV=docs
+    #PPC64LE
+  - arch: ppc64le
+    python: pypy3
+  - arch: ppc64le
+    python: 3.5
+  - arch: ppc64le
+    python: 3.6
+  - arch: ppc64le
+    python: 3.7
+  - &latest_py3_ppc
+    arch: ppc64le
+    python: 3.8
+  - <<: *latest_py3_ppc
+    env: LANG=C
+  - arch: ppc64le
+    python: 3.8-dev
+  - arch: ppc64le
+    python: 3.9-dev
+  - <<: *latest_py3_ppc
+    env: TOXENV=docs
   allow_failures:
   # suppress failures due to pypa/setuptools#2000
   - python: pypy3
+  - arch: ppc64le
+    python: pypy3
   - <<: *latest_py3
     env: TOXENV=docs
+  - <<: *latest_py3_ppc
+    env: TOXENV=docs
+
 
 cache: pip
 
+before_install:
+  - |
```

Comment:
Why is this change necessary only on this platform? Can you add a comment explaining why it's here and if possible what defect requires it?

Reply:
Travis base image for PPC64LE does not provide user level permission to ~/.cache/pip/wheels, result in build failure during pip install or tox trying to install packages. Therefore this is fix only for ppc64le and for travis build. Going forward, as and when fix will be made in build images, this fix would not create problem.

Reply:
@ezeeyahoo Can you address this one question, then we can merge?

Reply:
It is needed for PPC64LE only due to the fact that Travis linux base image for this platform lack proper permissions to the directory ~/.cache/pip/wheels that allows user running travis build to install pip packages which is otherwise allowed on linux images for amd64.

Would you like me to add inline comment through a new commit to the same PR?

Reply:
Yes, in the same PR. New commit or amend an existing commit.

Reply:
Ideally, this comment would also link to a bug filed with Travis that captures the need for this workaround so that someone stumbling on this code could find out if the workaround is still needed.

Reply:
Updated in d78027a25.